src/transaction_builder.js

```diff
@@ -16,6 +16,8 @@ var ECPair = require('./ecpair')
 var ECSignature = require('./ecsignature')
 var Transaction = require('./transaction')
 
+const debug = require('debug')('bitgo:utxolib:txbuilder')
```

Comment:
Change const to var.
For now, we are trying to keep it ES5 compatible since this was the library's original style.